internal/hcs/utils.go

```diff
@@ -1,10 +1,15 @@
 package hcs
 
 import (
+	"context"
 	"io"
 	"syscall"
 
 	"github.com/Microsoft/go-winio"
+	diskUtil "github.com/Microsoft/go-winio/vhd"
```

Comment:
Go convention is to keep import names all-lowercase. `diskutil` would be fine though.